content/monitors/faq/how-to-use-the-monitor-template-variables.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
This faq has been merged here: https://docs-staging.datadoghq.com/gus/redesign/monitors/notifications/#monitor-template-variable-explained could you remove it from your PR ? :)